### PR TITLE
Added basic slack notifications

### PR DESCRIPTION
**Pull request by sneub**
Simple notifications, untested code 😇

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+slack-webhook
```
Comment (crmne) [inline requirements.txt:1]: I don't use `requirements.txt` for this project, rather `pipenv`. Could it be possible to use `pipenv install slack-webhook` or put it in the `Pipfile` instead? Thank you :)

File: rfid_tracker/app.py [hunk 2]
```diff
@@ -7,10 +7,13 @@
 import typer
 from mfrc522 import SimpleMFRC522
 from models import Action, Base, Member
+from slack_webhook import Slack
 
 MAX_SPOTS = 19
+WEBHOOK_URL = 'https://hooks.slack.com/services/xxx/yyy'
```
Comment (crmne) [inline rfid_tracker/app.py:13]: I think this should be removed, and rather... (see next comment)

File: rfid_tracker/app.py [hunk 3]
```diff
@@ -7,10 +7,13 @@
...
 
 app = typer.Typer()
+slack = Slack(url=WEBHOOK_URL)
```
Comment (crmne) [inline rfid_tracker/app.py:16]: ... use environment variables for it:

`os.environ["WEBHOOK_URL"]`

File: rfid_tracker/app.py [hunk 4]
```diff
@@ -155,6 +158,7 @@ def tracker(
                     session.add(new_action)
                     session.commit()
                     bye(user_in_db.first_name, user_in_db.last_name)
+                    slack.post(text=f"{user_in_db.first_name} just checked out. {n_occupied_spots+1} spots left.")
```
Comment (crmne) [inline rfid_tracker/app.py:161]: We have multiple Dominiks for example, it would be nice to have both first name and last name :)

**Review by crmne — CHANGES_REQUESTED** [review]
Thanks for the changes! There are a few things to be changed before merging this.

File: rfid_tracker/app.py (hunk 4 above)
Comment (sneub) [reply rfid_tracker/app.py:161]: Good call!

File: rfid_tracker/app.py (hunk 3 above)
Comment (sneub) [reply rfid_tracker/app.py:16]: Done

File: rfid_tracker/app.py (hunk 2 above)
Comment (sneub) [reply rfid_tracker/app.py:13]: Done

File: requirements.txt (hunk 1 above)
Comment (sneub) [reply requirements.txt:1]: It seems I'm living in the past! Switching to pipenv pronto 🚀

**Pull request merged**